src/objective_types/oncedifferentiable.jl

```diff
@@ -99,72 +78,18 @@ function OnceDifferentiable(f, x_seed::AbstractArray, F::AbstractArray, DF::Abst
         fdfF = make_fdf(f, x_seed, F)
         return OnceDifferentiable(fF, dfF, fdfF, x_seed, F, DF)
     else
-        if is_finitediff(autodiff)
-            # Figure out which Val-type to use for FiniteDiff based on our
-            # symbol interface.
-            fdtype = finitediff_fdtype(autodiff)
-            # Apparently only the third input is aliased.
-            j_finitediff_cache = FiniteDiff.JacobianCache(copy(x_seed), copy(F), copy(F), fdtype)
-            if autodiff == :finiteforward
-                # These copies can be done away with if we add a keyword for
-                # reusing arrays instead for overwriting them.
-                Fx = copy(F)
-                DF = copy(DF)
-
-                x_f, x_df = x_of_nans(x_seed), x_of_nans(x_seed)
-                f_calls, j_calls = [0,], [0,]
```

Comment:
Should we try to preserve this call count? I don't think it makes a lot of sense for autodiff in general, because some backends will not go _through_ the actual code to compute the gradient (unlike ForwardDiff and FiniteDiff).

Reply:
In addition, I don't think this call count was present for every operator or every autodiff method

Reply:
I have to look. It's done this way because people complained that it didn't match with what they printed from their objective, but I agree that people will have to calculate the number themselves in the special case of finite diff